### DISCUSSION TITLE
How to setup validation on react-select Creatable input control?

### DISCUSSION THREAD

**Discussion by andre7000** [react-hook-form/react-hook-form#2038] [Q&A] [answered]
Hello!  Just starting out with react-hook-form!

1. How do you set a validation rule to require input on the react-select creatable input control and display an error message if there is no input on submission?  https://react-select.com/creatable

2. In addition, once the user either creates a new input value or selects an input value from the drop down, I'd like to update the state in React.

https://codesandbox.io/s/cold-bush-5ehgf?file=/src/index.js

Thank you for your hard work on react-hook-form and any help you can provide!

**Comment by bluebill1049** [comment, marked as answer]
I would recommend you to have read on this section: https://react-hook-form.com/api#Controller there is a codesandbox for react-select as well

**Reply by andre7000** [reply]
Looks like I just figured it out!  Thanks!

**Reply by Nirosh-cartrabbit** [reply]
Hii i have a issue in react select creatable, and i dont see any code sandbox related to it. i have issue in removing the newly added value in creatable. can u help me up?? @andre7000  @bluebill1049
